README.md

```diff
@@ -1,7 +1,8 @@
 # RubiX
 
 [![Build Status](https://travis-ci.org/qubole/rubix.svg?branch=master)](https://travis-ci.org/qubole/rubix)
-[![Coverage Status](https://coveralls.io/repos/github/qubole/rubix/badge.svg)](https://coveralls.io/github/qubole/rubix)
+[![codecov](https://codecov.io/gh/TechnionYP5777/project-name/branch/master/graph/badge.svg)](https://codecov.io/gh/qubole/rubix)
```

Comment:
The URL is wrong.. 
It should be `https://codecov.io/gh/qubole/rubix/graph/badge.svg`

Reply:
Done. Now it is properly showing codecov coverage.